migrations/src/main/scala/za/co/absa/enceladus/migrations/continuous/migrate01/MigratorDataset.scala

```diff
@@ -0,0 +1,152 @@
+/*
+ * Copyright 2018-2019 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.enceladus.migrations.continuous.migrate01
+
+import java.time.ZonedDateTime
+
+import org.apache.log4j.{LogManager, Logger}
```

Comment:
Let's use `slf4j` all around.

Reply:
Done for migrations. Raised #802 for other modules.